### ISSUE TITLE
Wrong:python2 $DENSEPOSE/detectron/tests/test_spatial_narrow_as_op.py

### ISSUE THREAD

**Issue by Anikily** [facebookresearch/Detectron#693]
## PLEASE FOLLOW THESE INSTRUCTIONS BEFORE POSTING
1. Please thoroughly read README.md, INSTALL.md, GETTING_STARTED.md, and FAQ.md
2. Please search existing *open and closed* issues in case your issue has already been reported
3. Please try to debug the issue in case you can solve it on your own before posting

## After following steps 1-3 above and agreeing to provide the detailed information requested below, you may continue with posting your issue
when install the DensePose in my machine,and use this command:Check that Detectron tests pass,python2 $DENSEPOSE/detectron/tests/test_spatial_narrow_as_op.py
,i encountered this problem:No handlers could be found for logger "caffe2.python.net_drawer"
net_drawer will not run correctly. Please install the correct dependencies.
E1005 15:08:03.076932 15809 init_intrinsics_check.cc:43] CPU feature avx is present on your machine, but the Caffe2 binary is not compiled with it. It means you may not get the full speed of your CPU.
Found Detectron ops lib: /home/tensor/anaconda2/envs/caffe2_env/lib/libcaffe2_detectron_ops_gpu.so
E.terminate called without an active exception
*** Aborted at 1538723284 (unix time) try "date -d @1538723284" if you are using GNU date ***
PC: @     0x7fa8b6f68428 gsignal
*** SIGABRT (@0x3e800003dc1) received by PID 15809 (TID 0x7fa8b820b700) from PID 15809; stack trace: ***
    @     0x7fa8b7a1e390 (unknown)
    @     0x7fa8b6f68428 gsignal
    @     0x7fa8b6f6a02a abort
    @     0x7fa860ec53df __gnu_cxx::__verbose_terminate_handler()
    @     0x7fa860ec3b16 __cxxabiv1::__terminate()
    @     0x7fa860ec3b4c std::terminate()
    @     0x7fa8124e4c70 caffe2::CUDAContext::~CUDAContext()
    @     0x7fa812510332 caffe2::SpatialNarrowAsOp<>::~SpatialNarrowAsOp()
    @     0x7fa85f165bc5 caffe2::Workspace::RunOperatorOnce()
    @     0x7fa8600fe0b8 _ZZN6caffe26python16addGlobalMethodsERN8pybind116moduleEENKUlRKNS1_5bytesEE26_clES6_.isra.3127.constprop.3187
    @     0x7fa8600fe264 _ZZN8pybind1112cpp_function10initializeIZN6caffe26python16addGlobalMethodsERNS_6moduleEEUlRKNS_5bytesEE26_bJS8_EJNS_4nameENS_5scopeENS_7siblingEEEEvOT_PFT0_DpT1_EDpRKT2_ENUlRNS_6detail13function_callEE1_4_FUNESQ_
    @     0x7fa8601336b0 pybind11::cpp_function::dispatcher()
    @     0x7fa8b7d0ccc4 PyEval_EvalFrameEx
    @     0x7fa8b7d0e4e9 PyEval_EvalCodeEx
    @     0x7fa8b7d0b9b8 PyEval_EvalFrameEx
    @     0x7fa8b7d0e4e9 PyEval_EvalCodeEx
    @     0x7fa8b7d0b9b8 PyEval_EvalFrameEx
    @     0x7fa8b7d0e4e9 PyEval_EvalCodeEx
    @     0x7fa8b7d0b9b8 PyEval_EvalFrameEx
    @     0x7fa8b7d0e4e9 PyEval_EvalCodeEx
    @     0x7fa8b7c97377 function_call
    @     0x7fa8b7c727a3 PyObject_Call
    @     0x7fa8b7d074be PyEval_EvalFrameEx
    @     0x7fa8b7d0e4e9 PyEval_EvalCodeEx
    @     0x7fa8b7c9728a function_call
    @     0x7fa8b7c727a3 PyObject_Call
    @     0x7fa8b7c8163d instancemethod_call
    @     0x7fa8b7c727a3 PyObject_Call
    @     0x7fa8b7ccb8a4 slot_tp_call
    @     0x7fa8b7c727a3 PyObject_Call
    @     0x7fa8b7d08b69 PyEval_EvalFrameEx
    @     0x7fa8b7d0e4e9 PyEval_EvalCodeEx
installing caffe2 with prebuild,so there is no build directory.Again,there is no $PYTHONPATH here.

### System information

* Operating system: ubontu 16.04
* Compiler version: ?
* CUDA version: cuda9.0
* cuDNN version: cuDNN7.0
* NVIDIA driver version: ?
* GPU models (for all devices if they are not all the same): ?
* `PYTHONPATH` environment variable: None
* `python --version` output: 2.7
* Anything else that seems relevant: ?

**Comment by Yansslc** [comment]
bd

**Comment by Anikily** [comment]
> bd

ok thanks so much my labmate

**Comment by phuccuongngo99** [comment]
Have you found a fixed?
What does bd mean?

**Comment by Anikily** [comment]
> Have you found a fixed?
> What does bd mean?

Yes.I have fixed this problem and the meaning of 'bd' is the abbreviation of chinese word‘帮顶’

**Comment by Anikily** [comment]
@phuccuongngo99
